testdata/p4_16_samples_outputs/gauntlet_various_ops-bmv2.p4-stderr

```diff
@@ -1,21 +1,18 @@
+gauntlet_various_ops-bmv2.p4(118): [--Wwarn=overflow] warning: 4s12: signed value does not fit in 4 bits
+        h.rshift.d = -4s7 >> 1 >> 1;
+                     ^^^^^^^^^
+gauntlet_various_ops-bmv2.p4(118): [--Wwarn=overflow] warning: 4s14: signed value does not fit in 4 bits
+        h.rshift.d = -4s7 >> 1 >> 1;
+                     ^^^^^^^^^^^^^^
```

Comment:
Something has gone awry here -- these messages are wrong (and different!)

The result of `-4s7 >> 2` should be `-4s2`